common/utils/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -99,7 +100,7 @@ case class MessageWithContext(message: String, context: java.util.HashMap[String
  * Companion class for lazy evaluation of the MessageWithContext instance.
  */
 class LogEntry(messageWithContext: => MessageWithContext) {
-  def message: String = messageWithContext.message
+  def message: String = StringEscapeUtils.unescapeJava(messageWithContext.message)
```

Comment:
Hi, @panbingkun and @gengliangwang .

This seems to break SparkR somehow. Could you take a look at this?

- https://github.com/apache/spark/actions/workflows/build_sparkr_window.yml

```
-- Error ('test_basic.R:25:3'): create DataFrame from list or data.frame -------
Error in `handleErrors(returnStatus, conn)`: java.lang.IllegalArgumentException: Unable to parse unicode value: serF
	at org.apache.commons.text.translate.UnicodeUnescaper.translate(UnicodeUnescaper.java:55)
	at org.apache.commons.text.translate.AggregateTranslator.translate(AggregateTranslator.java:58)
	at org.apache.commons.text.translate.CharSequenceTranslator.translate(CharSequenceTranslator.java:101)
	at org.apache.commons.text.translate.CharSequenceTranslator.translate(CharSequenceTranslator.java:63)
	at org.apache.commons.text.StringEscapeUtils.unescapeJava(StringEscapeUtils.java:802)
	at org.apache.spark.internal.LogEntry.message(Logging.scala:103)
```

Reply:
Okay, Let me investigate it

Reply:
@dongjoon-hyun
I have identified the root cause of this issue and am currently attempting to fix it
https://github.com/apache/spark/pull/46897#issuecomment-2177869329